index.html

```diff
@@ -0,0 +1,130 @@
+      </p>
+    </section>
+
+    <section class="flexbox-container">
+      <h2 class="flex-top">Selection of Dogs</h2>
+      <!-- W3 HTML code check states that a h2 header is necessary -->
+      <article class="flex-dog">
+        <img src="images/murphy-card.jpg" alt="Light brown dog, staring at something with curiosity." width="175" onclick="alert('Murphy, $123.45')">
+        <h2>Murphy</h2>
+        <h3><strong>Cost to Adopt:</strong> $123.45</h3>
+        <p>Lemon drops soufflé apple pie. Soufflé jelly cheesecake. I love jelly I love toffee sesame snaps tootsie roll cake.
+        </p>
+        <button onclick="addToTotal(123.45)">Adopt</button>
+      </article>
+
+      <article class="flex-dog">
+        <img src="images/poppy-card.jpg" alt="White/tan shih-tzu that's walking with a smile." width="175" onclick="alert('Poppy, $123.45')">
+        <h2>Poppy</h2>
+        <h3><strong>Cost to Adopt:</strong>$123.45</h3>
+        <p>Lemon drops soufflé apple pie. Soufflé jelly cheesecake. I love jelly I love toffee sesame snaps tootsie roll cake.
+        </p>
+        <button onclick="addToTotal(123.45)">Adopt</button>
+      </article>
+
+      <article class="flex-dog">
+        <img src="images/jack-card.jpg" alt="Beagle that's looking up at something to its right." width="175" onclick="alert('Jack, $123.45')">
+        <h2>Jack</h2>
+        <h3><strong>Cost to Adopt:</strong> $123.45</h3>
+        <p>TLemon drops soufflé apple pie. Soufflé jelly cheesecake. I love jelly I love toffee sesame snaps tootsie roll cake.</p>
+        <button onclick="addToTotal(123.45)">Adopt</button>
+      </article>
+
+      <article class="flex-dog">
+        <img src="images/duffy-card.jpg" alt="White and tan dog with blue eyes that's staring intently at the camera, pitbull." width="175" onclick="alert('Duffy, $123.45')">
+        <h2>Duffy</h2>
+        <h3><strong>Cost to Adopt:</strong> $123.45</h3>
+        <p>Lemon drops soufflé apple pie. Soufflé jelly cheesecake. I love jelly I love toffee sesame snaps tootsie roll cake.</p>
+        <button onclick="addToTotal(123.45)">Adopt</button>
+      </article>
+
+      <article class="flex-dog">
+        <img src="images/lucas-card.jpg" alt="Black dog with reddish/tan eyes, staring and smiling with it's mouth wide open." width="175" onclick="alert('Lucas, $123.45')">
+        <h2>Lucas</h2>
+        <h3><strong>Cost to Adopt:</strong> $123.45</h3>
+        <p>Lemon drops soufflé apple pie. Soufflé jelly cheesecake. I love jelly I love toffee sesame snaps tootsie roll cake.</p>
+        <button onclick="addToTotal(123.45)">Adopt</button>
+      </article>
+
+      <article class="flex-dog">
+        <img src="images/jake-card.jpg" alt="Yellow dog with a brown/tan collar." width="175" onclick="alert('Jake, $123.45')">
```

Comment:
Great! These certainly work, but see if you can create a function in the JS, that when called does the alert. Just a note for how to improve.